babel/src/main/java/org/apache/calcite/sql/babel/SqlTableAttributeBlockCompression.java

```diff
@@ -20,20 +20,20 @@
 import org.apache.calcite.sql.parser.SqlParserPos;
 
 /**
- * A <code>SqlCreateAttributeBlockCompression</code> is a CREATE TABLE option
+ * A <code>SqlTableAttributeBlockCompression</code> is a CREATE TABLE option
```

Comment:
nit: Change the comment to match the name change. It's more than a CREATE TABLE option.